common/src/app/common/pages/changes.cljc

```diff
@@ -28,15 +34,11 @@
   [coll o]
   (into [] (filter #(not= % o)) coll))
 
-(defn vec-without-nils
-  [coll]
-  (into [] (remove nil?) coll))
-
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Page Transformation Changes
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-;; --- Changes Processing Impl
+;; === Changes Processing Impl
```

Comment:
The `---` is used in almost all files, there are a reason to change it? I mean it is just for curiosity.

Reply:
Actually I don't remember well :smile:. I think it's something similar to markdown heading levels. `===` for *changes* and *operations* helpers, and `---` for *media*, *components* and *typographies*, that are kind of second level under *changes*.

But probably a big "fumada mental".